notebooks/Bulk Update Maps for Use in Collector or Field Maps.ipynb

```diff
@@ -11,34 +11,26 @@
    "cell_type": "markdown",
```

Comment:
<p>I think it's more obvious if all 3 examples just have <code>owner:{gis.users.me.username}</code>  in the query.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/field-maps-scripts/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>